src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -352,10 +352,29 @@ class GoogleExtensions(val directoryDAO: DirectoryDAO, val accessPolicyDAO: Acce
     } yield ()
   }
 
+  def getSynchronizedState(groupId: WorkbenchGroupIdentity): Future[Option[GroupSyncResponse]] = {
+    val groupDate = getSynchronizedDate(groupId)
+    val groupEmail = getSynchronizedEmail(groupId)
+
+    for {
+      date <- groupDate
+      email <- groupEmail
+    } yield {
+      if(email.isDefined || date.isDefined)
+        Option(GroupSyncResponse(date.getOrElse("").toString, email.getOrElse(WorkbenchEmail(""))))
```

Comment:
@MatthewBemis's comment got me looking a little more deeply. Changing the end point in this way is a subtle break in backwards compatibility. A response of None in the old way meant that the group was never synced. But in the new way it means that there is neither a sync data nor an email. But I can't think of a good way to introduce email here which makes me think this is the wrong place.

I see it is already in the list access policies call, how about also adding it to the load access policy (singular) call too?

Reply:
the thought here was to contain sync date and policy email to one endpoint that would live under the google path because they are google-related attributes. i originally thought about having it put in the endpoint that you're referring to but it felt slightly out of place to return google sync date in that endpoint.

another alternative i would propose is keeping things mostly the way they are in this PR except changing it to return None in the event of a missing sync date _or_ a missing email. that should ensure we keep backwards compatibility.

code would look something like

```
(dateOpt, emailOpt) match {
  case (Some(date), Some(email) => Option(GroupSyncResponse(date.toString, email))
  case _ => None
}
```

because we create the email attribute at policy creation, in practice we should never have a situation where the email is missing so really it would just be a missing sync date that dictates whether or not we return None, which is equivalent to what it does today.